test/AutoMapper.Extensions.Microsoft.DependencyInjection.Tests/Profiles.cs

```diff
@@ -29,7 +29,7 @@ public Profile1()
 
     public abstract class AbstractProfile : Profile { }
 
-    internal class Profile2 : Profile
+    public class Profile2 : Profile
```

Comment:
I made this public because `AddAutoMapper()` had different behavior compared to `IConfigurationExpression.AddProfiles()`. `AddProfiles()` only adds public Profiles.